packages/mysql2/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@sqlfx/mysql2",
+  "version": "0.45.0",
```

Comment:
If you can set the version to 0.0.0, then run `pnpm changeset` and add a minor changeset for the initial version.

Reply:
I see that the changeset version is literally "minor". Did you mean 0.0.1 instead? Sorry, I've never used changesets before.